test/coverlet.core.tests/Samples/Instrumentation.CatchBlock.cs

```diff
@@ -0,0 +1,55 @@
+﻿// Remember to use full name because adding new using directives change line numbers
+
+using System.Threading.Tasks;
+
+namespace Coverlet.Core.Samples.Tests
+{
+    public class CatchBlock
```

Comment:
I'd add more sample
* Multiple throw inside catch with some condition i.e.
```cs
if (bool) 
    throw; 
else
   throw new Exception();
```
* Nested try/catch
* Catch with Type `catch(DivideByZero)` and inside condition with `throw` and `throw Exception` single and nested